packages/icons/src/icon/index.js

```diff
@@ -1,25 +1,27 @@
 /**
  * WordPress dependencies
  */
-import { cloneElement } from '@wordpress/element';
+import { cloneElement, forwardRef } from '@wordpress/element';
 
 /** @typedef {{icon: JSX.Element, size?: number} & import('@wordpress/primitives').SVGProps} IconProps */
 
 /**
  * Return an SVG icon.
  *
- * @param {IconProps} props icon is the SVG component to render
- *                          size is a number specifiying the icon size in pixels
- *                          Other props will be passed to wrapped SVG component
+ * @param {IconProps}                          props icon is the SVG component to render
+ *                                                   size is a number specifiying the icon size in pixels
+ *                                                   Other props will be passed to wrapped SVG component
+ * @param {import('react').Ref<SVGSVGElement>} ref   The forwarded ref to the SVG element.
```

Comment:
Since `Icon` could be any JSX element, it's probably safer to go with `any` (we do this in almost every components in the package)

Also, we usually use the `ForwardedRef` type

```suggestion
 * @param {IconProps}                         props icon is the SVG component to render
 *                                                  size is a number specifiying the icon size in pixels
 *                                                  Other props will be passed to wrapped SVG component
 * @param {import('react').ForwardedRef<any>} ref   The forwarded ref to the SVG element.

Reply:
I use the `SVGSVGElement` type because the description of the `icon` prop says that it should render an `SVG` component. Do you think if it matters here? I've changed that to `HTMLElement` for now as I think it's probably better than `any`? 😆